desktop-app/src/renderer/components/ToolBar/AddressBar/index.tsx

```diff
@@ -241,7 +246,9 @@ const AddressBar = () => {
           >
             <Icon icon={isHomepage ? 'mdi:home' : 'mdi:home-outline'} />
           </Button>
-          <Bookmark currentAddress={address} />
+          {pageTitle && (
+            <Bookmark pageTitle={pageTitle} currentAddress={address} />
+          )}
```

Comment:
Is there any particular reason for the conditional check of `pageTitle` in this line? 
If not, then I think we can remove the conditional check and directly render the `<Bookmark pageTitle={pageTitle} currentAddress={address} />` component.

Reply:
@themohammadsa Currently there's no reason, so Yes we can remove that.
Thanks